---
layout: post
title: Exonic splice variant discovery using in vitro models of inherited retinal disease
original_url: https://linkinghub.elsevier.com/retrieve/pii/S2666-2477(24)00097-6
abstract: orrect identification of the molecular consequences of pathogenic genetic variants is
essential to the development of allele-specific therapies. However, such molecular
effects may remain ambiguous following genetic sequence analysis alone. Here, we
identify exonic codon-altering variants that are also predicted to disrupt normal RNA
splicing in the context of inherited retinal disease. NR2E3 c.932G>A (p.Arg311Gln) is a
variant commonly associated with Enhanced S Cone Syndrome (ESCS). Previous
studies using mutagenized cDNA constructs have shown that the arginine to glutamine
substitution at position 311 of NR2E3 does not meaningfully diminish function of the
rod-specific transcription factor. Using retinal organoids, we explored the molecular
consequences of NR2E3 c.932G>A when expressed endogenously during human rod
photoreceptor cell development. Retinal organoids carrying the NR2E3 c.932G>A allele
expressed a transcript containing a 186-nucleotide deletion of exon 6 within the ligand
binding domain. This short transcript was not detected in control organoids or control
human donor retina samples. A minigene containing exons 5 and 6 of NR2E3 showed
sufficiency of the c.932G>A variant to cause the observed splicing defect. These results
support the hypothesis that the pathogenic NR2E3 c.932G>A variant leads to
photoreceptor disease by causing a splice defect and not through an amino acid
substitution as previously supposed. They also explain the relatively mild effect of
Arg311Gln on NR2E3 function in vitro. We also used in silico prediction tools to show
that similar changes are likely to affect other inherited retinal disease variants in genes
such as CEP290, ABCA4, and BEST1.

tags:
  - genetics
---

